.github/PULL_REQUEST_TEMPLATE/create-a-geos-chem-pull-request.md

```diff
@@ -1,79 +1,31 @@
 ---
 name: Submit updates to GEOS-Chem with a pull request
-about: This template instructs users to submit a GEOS-Chem pull request
-title: "[PULL REQUEST]"
 labels: 'never stale'
-assignees: ''
 
 ---
 
-# GEOS-Chem Update Submission Form
-<!--- Please fill out this form to submit an update for consideration in GEOS-Chem.  Contact the [GEOS-Chem Support Team](http://wiki.geos-chem.org/GEOS-Chem_Support_Team) with any questions. -->
-<!--- Please also see our [Submitting updates for inclusion into GEOS-Chem](http://wiki.geos-chem.org/Submitting_updates_for_inclusion_in_GEOS-Chem) wiki page for more information. -->
+<!--- Before submitting a pull request please see:
```

Comment:
Same comment as for the issues templates.

Reply:
Now done in commit 581e361a